testssl.sh

```diff
@@ -2705,11 +2822,13 @@ run_hpkp() {
                "$has_backup_spki" && out "$spaces"       # we had a few lines with backup SPKIs already
                prln_svrty_high " No matching key for SPKI found "
                fileout "HPKP_SPKImatch" "HIGH" "None of the SPKI match your host certificate, intermediate CA or known root CAs. Bricked site?"
+               set_grade_cap "A" "Problems with HTTP Public Key Pinning (HPKP)"
```

Comment:
For a bricked site an A?

Reply:
According to the rules, yes: https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide#changes-in-2009l-4-march-2016

Again, i think we can adjust accordingly (F).